homeassistant/components/mopeka/__init__.py

```diff
@@ -22,11 +24,14 @@
 type MopekaConfigEntry = ConfigEntry[PassiveBluetoothProcessorCoordinator]
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: MopekaConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please revert the typing change on this line

Reply:
you may have an older version, mine shows ConfigEntry

Reply:
Cleared browser cache and it’s still there. Will double check it by pulling manually when I’m back at my desk

Reply:
no my bad, i thought i had changed that but guess i didn't rebase.